pkgs/tools/games/steamback/default.nix

```diff
@@ -0,0 +1,61 @@
+{ lib
+, buildPythonApplication
+, fetchPypi
+, pythonRelaxDepsHook
+, setuptools
+, pillow
+, psutil
+, async-tkinter-loop
+, timeago
+, platformdirs
+, sv-ttk
+}:
+
+buildPythonApplication rec {
+  pname = "steamback";
+  version = "0.3.6";
+  format = "pyproject";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-hvMPSxIfwwQqo80JCpYhcbVY4kXs5jWtjjafVSMrw6o=";
+  };
+
+  nativeBuildInputs = [
+    pythonRelaxDepsHook
+  ];
+
+  buildInputs = [
+    setuptools
```

Comment:
setuptools probably goes to nativeBuildInputs like pillow if it is run on the build machine